MW Proofreading edits (#59)
mongodb/docs-rust: source/fundamentals/crud/compound-operations.txt

   * - ``projection`` 
     - | The projection to use when returning results.
       | Type: ``Document``
       | Default: ``None``

   * - ``sort``
     - | The sorting order to use when returning results. By default, the driver
         returns documents in their natural order, or as they appear in
         the database. To learn more, see :manual:`natural order </reference/glossary/#std-term-natural-order>`
         in the Server manual glossary.
       | Type: ``Document``
       | Default: ``None``

   * - ``write_concern``

>      - | The write concern for the operation. If you don't set a write
>          concern, the operation inherits the one set for the collection.

Reviewer: S: the use of "one" is a little unclear, so could reword like this

```suggestion
     - | The write concern for the operation. If you don't set this option, the operation inherits the write concern set for the collection.
```

   * - ``array_filters`` 
     - | The set of filters specifying the array elements to which the

>          update applies to.

Reviewer: ```suggestion
         update applies.
```

       | Default: ``false``

   * - ``write_concern``

>      - | The write concern for the operation. If you don't set a write
>          concern, the operation inherits the one set for the collection.
>          To learn more about write concerns, see :manual:`Write Concern

Reviewer: S: same as above

```suggestion
     - | The write concern for the operation. If you don't set this option, the operation inherits the write concern set for the collection.
         To learn more about write concerns, see :manual:`Write Concern
```

Reviewer: Applies elsewhere too
